### PR TITLE
frontend: More improvements for Gamescope

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
# Description

- Print Gamescope command and script to easily troubleshoot issues
- Omit "Use" in UI

When running, the output will look like this:

```
14:15:13 (INFO) Running Gamescope command: '/app/utils/bin/gamescope -f -n --adaptive-sync -- /tmp/tmpg09vzi_7.sh' 
14:15:13 (INFO) /tmp/tmpg09vzi_7.sh contains: 

#!/usr/bin/env sh
/var/home/TheEvilSkeleton/.var/app/com.usebottles.bottles/data/bottles/runners/soda-7.0-7/bin/wine64 start /wait 'C:\Program Files\Notepad++\notepad++.exe'

```

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] UI

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [x] Locally

### CODE REVIEW

File: bottles/backend/wine/winecommand.py [hunk 1]
```diff
@@ -487,6 +487,10 @@ def get_cmd(self, command, post_script: str = None, return_steam_cmd: bool = Fal
 
                 # Update command
                 command = f"{self.__get_gamescope_cmd(return_steam_cmd)} -- {gamescope_run}"
+                logging.info(f"Running Gamescope command: '{command}'")
+                logging.info(f"{gamescope_run} contains:")
+                with open(gamescope_run, "r") as f:
+                    print(f"\n{f.read()}\n")
```
Comment (mirkobrombin) [inline bottles/backend/wine/winecommand.py:493]: what's the purpose of this print?

Comment (TheEvilSkeleton) [reply bottles/backend/wine/winecommand.py:493]: It's easier to debug and it shows exactly it's running.

Comment (mirkobrombin) [reply bottles/backend/wine/winecommand.py:493]: Can you replace it with `loggin` like other logs?

Comment (TheEvilSkeleton) [reply bottles/backend/wine/winecommand.py]: Done.

**Comment by TheEvilSkeleton** [comment]
@mirkobrombin it looks like this:

image

**Comment by mirkobrombin** [comment]
I think it's fine

**Pull request merged**